gradle/libs.versions.toml

```diff
@@ -9,36 +9,37 @@ targetSdk = "33"
 jdk-target = "1.8"
 jdk-toolchain = "17"
 
-androidx-activity = "1.6.1"
-androidx-appcompat = "1.6.1"
-androidx-benchmark = "1.2.3"
+androidx-activity = "1.8.2"
+androidx-appcompat = "1.7.0"
+androidx-benchmark = "1.3.3"
 androidx-cardview = "1.0.0"
 androidx-compose-compiler = "1.5.14"
 # see https://developer.android.com/jetpack/compose/bom/bom-mapping
 androidx-compose-bom = "2024.09.02"
 androidx-constraintlayout = "2.1.4"
-androidx-core = "1.12.0"
-androidx-fragment = "1.3.6"
+androidx-core = "1.13.1"
+androidx-fragment = "1.8.5"
 androidx-gridlayout = "1.0.0"
 androidx-lifecycle = "2.8.7"
-androidx-navigation = "2.4.0-alpha09"
+androidx-navigation = "2.8.4"
 androidx-paging = "3.0.1"
 androidx-profileinstaller = "1.2.0-alpha02"
 androidx-recyclerview = "1.3.2"
 androidx-room = "2.4.0-alpha04"
 androidx-savedstate = "1.2.1"
 androidx-startup = "1.1.0"
-androidx-test = "1.5.0"
+androidx-test = "1.6.1"
 androidx-test-espresso = "3.5.1"
 androidx-test-junit-ext = "1.1.5"
 androidx-test-runner = "1.5.2"
 androidx-test-truth-ext = "1.5.0"
-androidx-tracing = "1.1.0"
-androidx-transition = "1.4.1"
+androidx-tracing = "1.2.0"
+androidx-transition = "1.5.1"
+androidx-viewbinding = "8.1.2"
```

Comment:
we did not have a version for viewbinding as we used the `androidTools` version, though we were only using that version for the AGP version, we need to separate their tracking.